runtime/integration-tests/src/generic/cases/ethereum_transaction.rs

```diff
@@ -25,9 +25,9 @@ use crate::generic::{
 // 	}
 pub const TEST_CONTRACT_CODE: &str = "608060405234801561001057600080fd5b50610113806100206000396000f3fe6080604052348015600f57600080fd5b506004361060325760003560e01c8063c2985578146037578063febb0f7e146057575b600080fd5b603d605f565b604051808215151515815260200191505060405180910390f35b605d6068565b005b60006001905090565b600060db576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004018080602001828103825260098152602001807f6572726f725f6d7367000000000000000000000000000000000000000000000081525060200191505060405180910390fd5b56fea2646970667358221220fde68a3968e0e99b16fabf9b2997a78218b32214031f8e07e2c502daf603a69e64736f6c63430006060033";
 
-#[test_runtimes([development])]
+#[test_runtimes(all)]
 fn call<T: Runtime>() {
-	RuntimeEnv::<T>::default().parachain_state_mut(|| {
```

Comment:
Is this change really needed?

Reply:
The diff seem off. The new line is:

```rust
RuntimeEnv::<T>::default().state_mut(|_evm| {
```

Reply:
I referred the line wrong. But what I wanted to ask is, that change is needed if we do not need the `evm` variable?